holoviews/plotting/bokeh/renderer.py

```diff
@@ -71,11 +92,11 @@ class BokehRenderer(Renderer):
     backend_dependencies = {'js': CDN.js_files if CDN.js_files else tuple(INLINE.js_raw),
                             'css': CDN.css_files if CDN.css_files else tuple(INLINE.css_raw)}
 
-    comms = {'default': (JupyterComm, None),
-             'server': (Comm, None)}
-
     _loaded = False
 
+    # Define the handler for updating matplotlib plots
```

Comment:
Outdated/incorrect comment? Or does this really involve matplotlib somehow?

Reply:
No, definitely wrong, must be leftover from the very beginning of the bokeh interface!